includes/content-distribution/class-taxonomy-terms.php

```diff
@@ -0,0 +1,149 @@
+<?php
+/**
+ * Newspack Content Distribution Taxonomy Terms Handler
+ *
+ * @package Newspack
+ */
+
+namespace Newspack_Network\Content_Distribution;
+
+use Newspack_Network\Content_Distribution as Content_Distribution_Class;
+
+/**
+ * Class to filter the canonical URLs for distributed content.
+ */
+class Taxonomy_Terms {
+
+	const SEPARATOR = '|--|';
+
+	/**
+	 * Get post taxonomy terms for distribution.
+	 *
+	 * @param \WP_Post $post The post object.
+	 * @return array The taxonomy term data.
+	 */
+	public static function get_post_taxonomy_terms( \WP_Post $post ) {
+		$ignored_taxonomies = Content_Distribution_Class::get_ignored_taxonomies();
```

Comment:
non-blocking: We could move this here now that we have a dedicated class for taxonomies.